docs/topics/proc_deploying-an-existing-wildfly-swarm-application-to-openshift.adoc

```diff
@@ -0,0 +1,60 @@
+
+[id='deploying-an-existing-wildfly-swarm-application-to-openshift]
+= Deploying an existing {WildFlySwarm} application to OpenShift
+
+You can easily deploy your existing application to OpenShift using the Fabric8 Maven plugin.
+
+.Prerequisites
+
+* A {WildFlySwarm}{ndash}based application
+
+.Procedure
+
+. Add the following profile to the `pom.xml` file in the root directory of your application:
++
+--
+[source,xml,options="nowrap"]
+----
+<profile>
+  <id>openshift</id>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>io.fabric8</groupId>
+        <artifactId>fabric8-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>resource</goal>
+              <goal>build</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
```

Comment:
Not sure what this section does or if it should be here.

Reply:
That's something we'll need to verify and possibly update in the chapter on creating new app from scratch. I vaguely recollect that excluding the `webapp` generator used to be necessary for Swarm, but not sure if that still applies.